conf/bootstrap_apt

```diff
@@ -149,6 +145,10 @@ Package: php-common
 Pin: origin packages.sury.org
 Pin-Priority: 550
 
+Package: *
```

Comment:
This bit needs to go at the top. It pins the priority for all packages from the sury repo to `10` (meaning that can't be installed, unless you explicitly install them). Then the packages that we do want to install are pinned to `550`, meaning slightly higher priority than the default Debian packages.

If you needed to do this to get it to install cleanly, then there must be something that it's installing that the default pinning is blocking. I suspect that there is a specific dependency that won't install cleanly.

I'm happy to have a look myself at some point, but I'm pretty under the pump, so likely won't be this week.

Reply:
I went through and changed the code back, but I will have to go through and do more testing to get successful build with common still being correct.